packages/Box/Box.jsx

```diff
@@ -51,10 +50,11 @@ const Box = ({
     getClassName('padding', 'vertical', ySize),
     getClassName('margin', 'bottom', below),
     getBetweenClasses(between, inline),
-    dangerouslyAddClassName
+    dangerouslyAddClassName,
+    rest.className
```

Comment:
`className` should be destructured in props

Reply:
That was done to avoid eslint demanding prop validation. Since it's a pretty standard react prop, should we have this?